src/AggregateRootBehaviour/ConstructionBehaviour.php

```diff
@@ -52,5 +51,5 @@ public static function reconstituteFromEvents(AggregateRootId $aggregateRootId,
         return $aggregateRoot;
     }
 
-    abstract protected function apply(Event $event);
+    abstract protected function apply(object $event);
```

Comment:
There's an inline comment that needs updating, too:

https://github.com/EventSaucePHP/EventSauce/blob/974844da975c22e8a6eff9218cc4070e9abdc5f2/src/AggregateRootBehaviour/ConstructionBehaviour.php#L46

Reply:
Done.